src/export/copy_across_regions/check_region_support_snapshot_export_to_s3_function.py

```diff
@@ -0,0 +1,32 @@
+import os
+import constants
+import utility
+
+
+def lambda_check_region_support_snapshot_export_to_s3(event, context):
+    """check whether the region supports snapshot export to s3"""
+    region = os.environ['Region']
+    instance_id = event['identifier']
+    result = {'identifier': instance_id,
+              'task': event['task']}
+
+    if utility.supports_snapshot_export_region(region):
+        result['taskname'] = constants.EXPORT_SNAPSHOT_SUPPORT
+    elif snapshot_present_in_snapshot_export_to_s3_supporting_region(instance_id,
+                                                                     os.environ['ExportSnapshotSupportedRegion']):
+        result['taskname'] = constants.EXPORT_FROM_REGION_THAT_SUPPORTS_SNAPSHOT_EXPORT_TO_S3
+    else:
+        result['taskname'] = constants.COPY_SNAPSHOT
+
+    return result
+
+
+def snapshot_present_in_snapshot_export_to_s3_supporting_region(instance_id, region):
```

Comment:
can this be more concise

Reply:
I am all for being as concise as possible to the point where you don't loose information. The result is what you see. I don't believe it can be shorter. Can you give me an example without loosing the essence? Otherwise let's keep it as it is.

Reply:
@stationeros 
- "The method returns a boolean so ideally should start with "is"" 
This I agree with. Will apply this part.
- "Also "export_to_s3" is something the method isn't doing."
The part export_to_s3 doesn't refer to exporting. You should read the whole part "snapshot_export_to_s3_supporting_region". It refers to a region_that_supports_snapshot_export_to_s3. I can change it to this as well, but thought mine had less words. 
If I place os.environ['ExportSnapshotSupportedRegion'] inside the method, so less parameters, than the name would need to be is_snapshot_available_in_region_that_supports_snapshot_export_to_s3 , though I am going to leave os.environ['ExportSnapshotSupportedRegion'] where it is as you didn't mention anything about this part, and change the method name to is_snapshot_available . ["2 hard things in computer science caching and naming things."](https://martinfowler.com/bliki/TwoHardThings.html) :)